.github/workflows/test.yml

```diff
@@ -21,6 +18,9 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      ELASTICSEARCH_AUTH_USERNAME: elastic
+      ELASTICSEARCH_AUTH_PASSWORD: changeme
```

Comment:
Do the tests run the compose file and run against that server? If so I don't think this is necessary is it?

Reply:
I'm confused. How do GHA env vars become available for tests that aren't running in GHA?

Reply:
These comments are talking about the GHA env vars...?

Reply:
Yes my mistake. The tests fail without the creds. Do you want me to remove them to make sure that they fail without these creds?

Reply:
There they just failed. Reverting back to the correct creds.